File: RelacionaWEB/Site/src/java/model/Banco.java
package model;
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */



import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.ResultSet;

/**
  *
  * @author Mathias
  */
public class Banco {

     public static Connection conexao = null;
     public PreparedStatement comando = null;
     public ResultSet tabela = null;
     
     public Banco() throws Exception {
         try {
             Class.forName("org.postgresql.Driver");
             if ((conexao == null) || (conexao.isClosed())) {
                 conexao =
DriverManager.getConnection("jdbc:postgresql://127.0.0.1:5432/WEBTrab2","postgres", "ifsp");
             }
         } catch (Exception ex) {
             throw new Exception("Erro de conexao:" + ex.getMessage());
         }
     }
}

File: RelacionaWEB/NodeJS/src/model/Banco.js
const { Client } = require('pg');

class Banco {
  static conexao;
  static init() {
    try {
      this.conexao = new Client({
        host: '127.0.0.1',
        port: 5432,
        database: 'WEBTrab2',
        user: 'postgres',
        password: 'ifsp',
      });
      this.conexao.connect();
    }
    catch (erro) {
      console.log('');
      console.log('==========================================================');
      console.log('');
      console.log(" >> ❌ Erro de conexao: " + erro + " ❌ << ");
      console.log('');
      console.log('==========================================================');
      console.log('');
    }
  }
}
module.exports = Banco;